src/kw_env.sh

```diff
@@ -216,11 +216,15 @@ function destroy_env()
   if [[ $(ask_yN 'Are you sure you want to delete this environment?') =~ 0 ]]; then
     return 22 # EINVAL
   fi
-  # Here it is checked if the current env is equal to the env we want to
-  # destroy, if so, we call the exit_env function to exit the environment mode.
-  current_env=$(< "${local_kw_configs}/${ENV_CURRENT_FILE}")
-  if [[ "$current_env" == "$env_name" ]]; then
-    exit_env
+  # Here it is checked if the env.current file exists,
+  # that is, if the user uses a current env.
+  if [[ -f "${local_kw_configs}/${ENV_CURRENT_FILE}" ]]; then
+    current_env=$(< "${local_kw_configs}/${ENV_CURRENT_FILE}")
+    # Here it is checked if the current env is equal to the env we want to
+    # destroy, if so, we call the exit_env function to exit the environment mode.
```

Comment:
You don't need all of these comments; just a single comment + a good commit message should be enough.

Reply:
ok!!